static/app/components/feedback/feedbackOnboarding/feedbackWidgetBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import type {CSSProperties} from 'react';
+
+import replaysDeadRageBackground from 'sentry-images/spot/replay-dead-rage-changelog.svg';
+
+import PageBanner from 'sentry/components/alerts/pageBanner';
+import {Button} from 'sentry/components/button';
+import {useFeedbackOnboardingSidebarPanel} from 'sentry/components/feedback/useFeedbackOnboarding';
+import {t} from 'sentry/locale';
+import useDismissAlert from 'sentry/utils/useDismissAlert';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export default function FeedbackWidgetBanner({style}: {style?: CSSProperties}) {
+  const {activateSidebar} = useFeedbackOnboardingSidebarPanel();
+  const organization = useOrganization();
+
+  const {dismiss, isDismissed} = useDismissAlert({
+    key: `${organization.id}:feedback-whats-new-banner`,
```

Comment:
is it intentional that this key is the same as in `FeedbackWhatsNewBanner`?

Reply:
ahhh copy and paste, not intentional